### PR TITLE
Refactor contract names

### PR DESCRIPTION
**Pull request by rjkz808**
Refactored [contracts](https://github.com/cyberacademia/cyberevents/tree/3-contract-names/contracts) and [tests](https://github.com/cyberacademia/cyberevents/tree/3-contract-names/test) names
See the issue #3

### CODE REVIEW

File: contracts/Event.sol [hunk 1]
```diff
@@ -98,15 +87,17 @@ contract CyberCore is CyberCoin {
     public 
     view 
     returns (
+      uint cashbackPercent,
```
Comment (mastercyb) [inline contracts/Event.sol]: It is not a name refactoring. Try to do atomic commits.

File: contracts/Event.sol [hunk 2]
```diff
@@ -189,12 +180,24 @@ contract CyberCore is CyberCoin {
...
+          events[eventId(_tokenId)].ticketPrice.div(100).
+          mul(events[eventId(_tokenId)].cashbackPercent)
+        )
+      );
+      paidAmountOf[ownerOf(_tokenId)][eventId(_tokenId)] = (
+        paidAmountOf[ownerOf(_tokenId)][eventId(_tokenId)].
+        sub(
+          events[eventId(_tokenId)].ticketPrice.div(100).
+          mul(events[eventId(_tokenId)].cashbackPercent)
+        )
```
Comment (mastercyb) [inline contracts/Event.sol]: Add a lot of business logic that is not connected to declared issue and pl topic

**Review by mastercyb — DISMISSED** [review]
Please split pull requests to (at least) 2 atomic changes

**Review by mastercyb — APPROVED** [review]

**Review by DearNadia — APPROVED** [review]
Cyberevents core - cyberacademia core?

**Pull request merged**